snap/snapenv/snapenv.go

```diff
@@ -113,16 +128,60 @@ func userEnv(info *snap.Info, home string) map[string]string {
 	return result
 }
 
-// envMap creates a map from the given environment string list, e.g. the
-// list returned from os.Environ()
-func envMap(env []string) map[string]string {
+// Environment variables glibc strips out when running a setuid binary.
+// Taken from https://sourceware.org/git/?p=glibc.git;a=blob_plain;f=sysdeps/generic/unsecvars.h;hb=HEAD
+var unsafeEnv = map[string]bool{
```

Comment:
This would be a nice thing to go generate eventually. No need to do it now but maybe add a comment.